### DISCUSSION TITLE
hi.CategoryTree and ModelsTree components keep flickering .

### DISCUSSION THREAD

**Discussion by shenshengyi** [iTwin/itwinjs-core#1242] [Q&A] [answered]
When I use the CategoryTree and ModelsTree components in itwin.js, I found that the flashing keeps on, is there any way to stop the flashing? Thanks.

**Comment by grigasp** [comment, marked as answer]
We don't see this behavior in other applications that use these components. Is it possible that you're causing re-renders by passing new (different) props? Can you record how the flickering looks like and upload the video?

**Comment by aruniverse** [comment]
Im assuming you used the itwin-viewer cra template, and youre seeing the same flickering issue described here: https://github.com/imodeljs/viewer-components-react/discussions/105. If so, should be as simple as updating the ver of tree-widget-react consumed in your app.

**Reply by shenshengyi** [reply]
Thank you very much, it has been resolved, follow the prompts.
